docs/conf.py

```diff
@@ -12,13 +12,12 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
-import sys
-sys.path.insert(0, os.path.abspath('../datadog_checks_base'))
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('../datadog_checks_base'))
+# sys.path.insert(1, os.path.abspath('../datadog_checks_dev'))
```

Comment:
Why the comments?

Reply:
I just put these here in case someone didn't want to install everything they could uncomment these and still have docs mostly build.